src/em/em.cpp

```diff
@@ -517,6 +517,40 @@ dm_sta_t *em_t::find_sta(mac_address_t sta_mac, bssid_t bssid)
     return NULL;
 }
 
+// Used to convert freq band from Onewifi to IEEE-1905-1-2013 table 6-23 & IEEE-1905-1-2013 table 6-23 to Wi-Fi Simple Configuration Technical Specification v2 table 44
+em_freq_band_t em_t::convert_freq_band(em_freq_band_t band)
+{
+    unsigned int int_band;
+    int_band = (int)band;
+
+    if (int_band == 1) {
+        return em_freq_band_24; // Return 2.4 GHz band
+    } else if (int_band == 2) {
+        return em_freq_band_5;  // Return 5 GHz band
+    } else if (int_band == 32 || band == 4) {
+        return em_freq_band_60;	// Return 60 GHz band
+    } else {
+        return em_freq_band_unknown; // Return unknown for other values
+    }
+}
+
+// Used to convert freq band from Wi-Fi Simple Configuration Technical Specification v2 table 44 to IEEE-1905-1-2013 table 6-23
+em_rd_freq_band_t em_t::map_freq_band_to_rf_band(em_freq_band_t band)
+{
+    unsigned int int_band;
+    int_band = (int)band;
+
+    if (int_band == 0) {
+        return em_rd_freq_band_24; // Return 2.4 GHz band
+    } else if (int_band == 1) {
+        return em_rd_freq_band_5; // Return 5 GHz band
+    } else if (int_band == 2) {
+        return em_rd_freq_band_60; // Return 60 GHz band
+    } else {
+        return em_rd_freq_band_unknown; // Return unknown for other values
+    }
+}
+
```

Comment:
In both of the above functions, 6G should be taken into consideration.

Reply:
6 GHz seems to be not specified in the specification : Table 6-23—AutoconfigFreqBand TLV (EasyMesh v5 spec) , Table 44 – RF Bands( WSC specification v2)